test/e2e/utils.go

```diff
@@ -28,10 +28,10 @@ var (
 	storageNamespace     = os.Getenv("STORAGE_NAMESPACE")
 	esServerUrls         = "http://elasticsearch." + storageNamespace + ".svc:9200"
 	cassandraServiceName = "cassandra." + storageNamespace + ".svc"
-    ctx 					*framework.TestCtx
-    fw 						*framework.Framework
-    namespace 				string
-    t 						*testing.T
+	ctx 					*framework.TestCtx
+	fw 						*framework.Framework
+	namespace 				string
+	t 						*testing.T
```

Comment:
Was this after running `make format`? Seems strange, unless it is having problems as these vars don't have `=`

Reply:
@objectiser Yes, it was after make format.  Sorry I didn't check before I submitted it, but I think I've gotten it straightened out now.